src/components/SearchResults.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export function SearchResults({ results, onSearchResultClick }) {
+  return (
+    <>
+      <div className="maps-location-search-results-container">
+        {results.filter(Boolean).map((result) => {
```

Comment:
inside array of object results can be null, because there are only two predictions on the user search. 

![Screen Shot 2022-08-29 at 11 43 41](https://user-images.githubusercontent.com/52234049/187124747-0340dc01-18b5-439a-befb-6ad95cbc462c.png)